Makefile

```diff
@@ -127,6 +132,10 @@ eslint-branch: node_modules/eslint node_modules/eslint-plugin-react node_modules
 mixedindentlint: node_modules/mixedindentlint
 	@echo "$(JS_FILES)\n$(SASS_FILES)" | xargs $(NODE_BIN)/mixedindentlint --ignore-comments --exclude="client/config/index.js"
 
+# Ensure that default config values exist in _shared.json
+config-defaults-lint: $(CONFIG_FILES)
+	@if [[ ! -z "$$(FORCE_COLOR=1 node server/config/validate-config-keys.js)" ]]; then exit 1; fi;
```

Comment:
Looks like this will exit with non-zero code if there is any output from `validate-config-keys.js`, which sounds about right. Otherwise there might be a more direct way to check the status code of the result of executing the script. I'm not familiar with `FORCE_COLOR` but I'd guess that was needed to allow `chalk` colors to come across?